XamarinCommunityToolkit.UnitTests/Converters/StringToDoubleConverter_tests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Globalization;
+using Xamarin.CommunityToolkit.Converters;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests.Converters
+{
+	public class StringToDoubleConverter_tests
+	{
+		[Theory]
+		[InlineData("2.2", 2.2)]
+		[InlineData("3.9", 3.9)]
+		[InlineData("255.98", 255.98)]
+		[InlineData("99191.98765", 99191.98765)]
+		public void StringToDoubleConverter(string value, double expectedResult)
+		{
+			var stringToDoubleConverter = new StringToDoubleConverter();
+
+			var result = stringToDoubleConverter.Convert(value, typeof(StringToIntConverter_tests), null,
```

Comment:
I think you mean `typeof(StringToDoubleConverter_tests)` instead of `typeof(StringToIntConverter_tests)` also for the test below